README.markdown

```diff
@@ -2,7 +2,7 @@ Resque
 ======
 
 [![Gem Version](https://badge.fury.io/rb/resque.svg)](https://rubygems.org/gems/resque)
-[![Build Status](https://travis-ci.org/resque/resque.svg)](https://travis-ci.org/resque/resque)
```

Comment:
It'll be like:

[![Build Status](https://github.com/mtsmfm/resque/actions/workflows/ci.yml/badge.svg)](https://github.com/mtsmfm/resque/actions/workflows/ci.yml)

Reply:
That's great! Thanks for adding the GH badge.